src/components/Notepad/Notepad.js

```diff
@@ -0,0 +1,27 @@
+import "./notepad.css";
+import React from "react";
+export const Notepad = (props) => {
```

Comment:
this is also correct , if possible you can try  direct "{ item }" instead of props